runtime-scientific/ucx/autobuild/defines

```diff
@@ -0,0 +1,26 @@
+PKGNAME=ucx
+PKGSEC=libs
+PKGDEP="rdma-core fuse-3"
+BUILDDEP__AMD64="${BUILDDEP} cuda valgrind"
+BUILDDEP__ARM64="${BUILDDEP} cuda valgrind"
+PKGDES="A communication framework for high-bandwidth and low-latency networks"
+PKGSUG="cuda"
+
+NOSTATIC=0
```

Comment:
Why?

Reply:
Don't know why. I copied that from openmpi spec